spec/jarvis/location_times_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "jarvis/location_times"
+
+describe Jarvis::LocationTimes do
+  let(:utc) { Time.new(2016, 1, 8, 16, 30, 00).utc }
```

Comment:
Looks like Time.new can take a utc_offset. so I can do this:

```
>> Time.new(2016, 1, 8, 16, 30, 00, 0).utc
=> 2016-01-08 16:30:00 UTC
```

(Add a final argument of `0`) and it seems to work.